src/olympia/editors/views.py

```diff
@@ -650,6 +651,14 @@ def review(request, addon, channel=None):
         except AddonApprovalsCounter.DoesNotExist:
             pass
 
+    reports = None
+    user_reviews = None
+    if addon.has_listed_versions():
```

Comment:
come to think of it, the post review stuff can be skipped for the unlisted review page too (none of the versions will be have been approved in that way)